src/idiomorph.js

```diff
@@ -208,7 +205,7 @@ var Idiomorph = (function () {
       // innerHTML, so we are only updating the children
       morphChildren(normalizedNewContent, oldNode, ctx);
       if (ctx.config.twoPass) {
-        restoreFromPantry(oldNode, ctx);
+        ctx.pantry.remove();
```

Comment:
when two pass is off ctx.pantry is just a dis joined div so we can just call remove() on it every time to simplify the code